templates/pg_hba.conf.j2

```diff
@@ -34,6 +34,13 @@ host  all  all  {{host}}  md5
 {% for host in postgresql_pg_hba_passwd_hosts %}
 host  all  all  {{host}}  password
 {% endfor %}
+# Password hosts per user
```

Comment:
@mprenditore If you want to do password authentication for a specific user, then you could use either `postgresql_pg_hba_default` or `postgresql_pg_hba_custom`

  ```postgresql_pg_hba_custom:
    - comment: "demo_user, password access"
      type: host
      database: all
      user: demo_user
      address: 0.0.0.0/0
      method: password```

Is there a specific reason we need this, as we don't to per-user access for the `md5` or `trust` method:

```# MD5 hashed password hosts
{% for host in postgresql_pg_hba_md5_hosts %}
host  all  all  {{host}}  md5
{% endfor %}

# Password hosts
{% for host in postgresql_pg_hba_passwd_hosts %}
host  all  all  {{host}}  password
{% endfor %}

# Trusted hosts
{% for host in postgresql_pg_hba_trust_hosts %}
host  all  all  {{host}}  trust
{% endfor %}```

Personally I don't use any of these sections, as it's too broad-brush.  I tend to always use `postgresql_pg_hba_default`, and `postgresql_pg_hba_custom`.

Reply:
Also, we probably don't want to be encouraging the use of `password`, as postgresql.org warn against it:

https://www.postgresql.org/docs/current/auth-password.html

_"It should always be avoided if possible."_